woocommerce/woocommerce-android
# Shipping Label Payments: part 1

First part of #3536, this adds the ability to see the current payment settings during the label creation process.

<img width=360 src="https://user-images.githubusercontent.com/1657201/107756395-73900280-6d24-11eb-88c1-575374705e48.png"/> <img width=360 src="https://user-images.githubusercontent.com/1657201/107756401-7559c600-6d24-11eb-8530-c8e7554fa91c.png"/>

#### Testing
1. Prepare an order that satisfies the shipping labels creation conditions.
2. Open the order detail in the app.
3. Click on Create shipping label.
4. Pass all the first steps until you reach the payment step.
5. Click on continue.
6. Confirm that the list payment methods are displayed, and that the settings match what you see in wp-admin.
7. Make some changes and confirm that the done button is displayed (the done button does nothing for now).

Update release notes:
